docs/config-description.md

```diff
@@ -1,452 +1,600 @@
 # Clio Config Description
-This file lists all Clio Configuration definitions in detail.
+
+This document provides a list of all available Clio configuration properties in detail.
+
+> [!NOTE]
+> Dot notation in configuration key names represents nested fields. For example, **database.scylladb** refers to the _scylladb_ field inside the _database_ object. If a key name includes "[]", it indicates that the nested field is an array (e.g., etl_sources.[]).
 
 ## Configuration Details
 
-### Key: database.type
+### database.type
+
 - **Required**: True
 - **Type**: string
-- **Default value**: cassandra
-- **Constraints**: The value must be one of the following: `cassandra`
- -   **Description**: Type of database to use. We currently support Cassandra and Scylladb. We default to Scylladb.
-### Key: database.cassandra.contact_points
+- **Default value**: `cassandra`
+- **Constraints**: The value must be one of the following: `cassandra`, `scylladb`.
+- **Description**: Specifies the type of database to use for storing and retrieving data needed by the Clio server. Supported options are Cassandra and ScyllaDB. If you don't provide a value, the Clio server uses ScyllaDB by default.
+
+### database.cassandra.contact_points
```

Comment:
If you set `database.type` to `scylladb`, all the keys below must also be `database.scylladb`. It would be good to reflect that somehow.